xdsl/dialects/linalg.py

```diff
@@ -1098,7 +1098,7 @@ def hidden_region(args: tuple[BlockArgument, ...]) -> None:
         )
 
 
-class PoolingOpsBase(NamedOpBase, ABC):
+class PoolingOpsBase(NamedOperation, ABC):
```

Comment:
```suggestion
class PoolingOperation(NamedOperation, ABC):
```
or change the PR title

Reply:
Oh it's because one had OpBase, the others OpsBase, this is one reason they all need to be Operation.